README.md

```diff
@@ -301,15 +301,23 @@ jobs:
       uses: actions/setup-python@v1
       with:
         python-version: 3.8
+
+#    Needed if the plugin is using Transifex, to have the lrelease command
+#    - name: Install Qt lrelease
+#      run: sudo apt-get install qt5-default qttools5-dev-tools
```

Comment:
```suggestion
#      run: sudo apt-get install qttools5-dev-tools
```

Reply:
do we need sudo on Travis?

Reply:
First, I didn't add `qt5-default`  ,but it was needed.

`could not find a Qt installation of ''`

CF https://stackoverflow.com/questions/16607003/qmake-could-not-find-a-qt-installation-of

> do we need sudo on GH?

Yes as well. It's password less.
https://stackoverflow.com/questions/57982945/how-to-apt-get-install-in-a-github-action

Reply:
ok!